### PR TITLE
fix: contrib-tests

### PR DESCRIPTION
**Pull request by Hk669**
## Why are these changes needed?

To verify that the `ContribTests` workflow is executing properly.

## Related issue number

addressed #2453 

## Checks

- [ ] I've included any doc changes needed for https://microsoft.github.io/autogen/. See https://microsoft.github.io/autogen/docs/Contribute#documentation to build and test documentation locally.
- [ ] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [ ] I've made sure all auto checks have passed.

### CODE REVIEW

File: .github/workflows/contrib-tests.yml [hunk 1]
```diff
@@ -45,12 +45,17 @@ jobs:
           pip install .[retrievechat-qdrant]
       - name: Install unstructured when python-version is 3.9 and on linux
         run: |
+          if [[ ${{matrix.os}} != ubuntu-latest ]]; then
```
Comment (ekzhu) [inline .github/workflows/contrib-tests.yml:48]: Why not use `runs-on: ubuntu-latest`

Comment (Hk669) [reply .github/workflows/contrib-tests.yml:48]: it disrupts the intended functionality

Comment (ekzhu) [reply .github/workflows/contrib-tests.yml:48]: Feel like we might want to separate the actions for different contrib tests. Changes made to one contrib class shouldn't need to trigger all contrib tests. @sonichi

Comment (Hk669) [reply .github/workflows/contrib-tests.yml:48]: yeah, this should work! @ekzhu

**Review by thinkall — CHANGES_REQUESTED** [review]
Please hold this PR untill #2455 is merged. PR #2439 broke RetrieveChatTests, and #2455 will fix it.

**Comment by Hk669** [comment]
> Please hold this PR untill #2455 is merged. PR #2439 broke RetrieveChatTests, and #2455 will fix it.

yeah sure!

**Review by thinkall — COMMENTED** [review]
Do we still need these changes?

**Comment by Hk669** [comment]
> Do we still need these changes?

nope we can close this PR.

**Pull request closed without merging**